### DISCUSSION TITLE
How does blobsaver work? (Q&A + Idea)

### DISCUSSION THREAD

**Discussion by r1sk01** [airsquared/blobsaver#494] [Ideas]
I'm writing a batch script for TSSChecker and would like to know how blobsaver grabs all the signed versions (betas and releases) and how to replicate how it works in batch with TSSChecker, or maybe there is some way to add command line utils to blobsaver?

**Comment by airsquared** [comment]
I would like to add a CLI mode to blobsaver (see #291), I just haven’t gotten to it yet. It would also be great if anyone was willing to help with that, however.

**Reply by r1sk01** [reply]
You're looking at the right person for the job, just tell me where everything is done from TSSChecker (like stealing the APNonce etc from the device and getting the blobs and versions, the whole ordeal) and I'll get it done!
(my discord is LunarNova#4471 if you'd like to talk)

**Reply by airsquared** [reply]
Sent you a friend request.
